src/3rd_party/CLI/ConfigFwd.hpp

```diff
@@ -117,7 +117,7 @@ class ConfigINI : public Config {
         std::vector<ConfigItem> output;
 
         while(getline(input, line)) {
-            std::vector<std::string> items;
+            std::vector<std::string> sitems;
```

Comment:
Yes, `items` is already defined in the parent class and is hidden by this declaration. I added the prefix `s` for `string`, because … I had to find something … I could have renamed it `items1` or so.